bugsnag-android-core/src/main/java/com/bugsnag/android/ThreadInternal.kt

```diff
@@ -18,6 +19,7 @@ class ThreadInternal internal constructor(
         writer.name("id").value(id)
         writer.name("name").value(name)
         writer.name("type").value(type.desc)
+        writer.name("state").value(state.name)
```

Comment:
Double-checking - will `getName()` be safe if ProGuard obfuscation is enabled?

Reply:
Progaurd doesn't obfuscate enum names by default - but that doesn't mean other tools won't. I'll sort out something more robust 👍